.eslintrc.json

```
@@ -1,11 +1,21 @@
 {
-  "extends": ["next/core-web-vitals", "plugin:storybook/recommended"],
+  "extends": ["next/core-web-vitals", "plugin:storybook/recommended", "plugin:jsx-a11y/recommended"],
   "plugins": ["unused-imports", "@typescript-eslint"],
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "project": true
   },
   "rules": {
+    "jsx-a11y/label-has-associated-control": "warn",
+    "jsx-a11y/anchor-has-content":"warn",
+    "jsx-a11y/role-supports-aria-props":"warn",
+    "jsx-a11y/no-noninteractive-element-to-interactive-role": "warn",
+    "jsx-a11y/click-events-have-key-events": "Warn",
+    "jsx-a11y/no-static-element-interactions": "warn",
+    "jsx-a11y/no-autofocus": "warn",
+    "jsx-a11y/img-redundant-alt":"warn",
+    "jsx-a11y/mouse-events-have-key-events": "warn",
+    "jsx-a11y/iframe-has-title": "warn",
```

Looks like these rules need warnings still as well.

```suggestion
    "jsx-a11y/iframe-has-title": "warn",
    "jsx-a11y/no-noninteractive-tabindex": "warn",
    "jsx-a11y/anchor-is-valid": "warn",
    "jsx-a11y/no-noninteractive-element-interactions": "warn",
    "jsx-a11y/interactive-supports-focus": "warn",    
```